__tests__/pages/edukasi.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import EducationPage, { getStaticProps } from "~/pages/edukasi";
+import educations from "~/lib/content/educations";
+
+jest.mock("~/lib/content/educations");
+jest.mock("next/router", () => require("next-router-mock"));
+
+describe("EducationPage", () => {
+  const { topics } = educations;
+
+  it("renders the title, description and breadcrumbs correctly", async () => {
+    render(<EducationPage educations={educations} />);
+
+    const title = screen.getByRole("heading", {
+      name: /Laman Edukasi Covid-19/i,
+    });
+    expect(title).toBeVisible();
+
+    const description = screen.getByText(
+      /Kumpulan informasi terbaru COVID-19, tips menjaga kesehatan selama pandemi, vaksinasi, hingga penanganan jika Anda terkonfirmasi positif COVID-19./i,
+    );
+    expect(description).toBeVisible();
+
+    const breadcrumbs = screen.getByRole("link", {
+      name: /Laman Edukasi Covid-19/i,
+    });
+    expect(breadcrumbs).toBeVisible();
+    expect(breadcrumbs).toHaveAttribute("href", "/edukasi");
+  });
+
+  it("renders the topics correctly", () => {
+    topics.forEach((topic) => {
+      expect(screen.getByText(topic.topic)).toBeVisible();
+
+      topic.contents.forEach((content) => {
+        const link = screen.getByTestId(`next-link-${content.title}`);
```

Comment:
Data-test-id should be our last priority when using selector in unit test.
It introduces additional attribute to the user.

Reply:
actually, it's copying from another test and I forgot to render the component on this case

Reply:
<img width="365" alt="Screen Shot 2021-08-13 at 06 29 39" src="https://user-images.githubusercontent.com/45778229/129282679-6303d62f-ae97-4f79-9d69-83b971cc1de9.png">

I will move the test into the components, but I have a little problem basically it's because the content has space (it's throwing error)

Reply:
I can fix from CMS

Reply:
forget it, the content has been updated, thanks